app/controllers/v1/greetings_controller.rb

```diff
@@ -0,0 +1,8 @@
+class V1::GreetingsController < ApplicationController
+  def index
+    @greetings = Greeting.find(Greeting.pluck(:id).sample)
```

Comment:
- [ ] Your app is rendering a static screen, and hence no random greeting is being rendered when the _greet_ button is clicked or the page is refreshed. Kindly ensure to fix this issue

Reply:
Thank you very much:+1:. I had a CORS issue but it is solved now.